python/tvm/ir/base.py

```diff
@@ -0,0 +1,86 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""Span data structures."""
+import tvm._ffi
+
+from tvm.runtime import Object
+from . import _ffi_api
+
+
+class Node(Object):
+    """Base class of all IR Nodes, implements astext function."""
+    def astext(self, show_meta_data=True, annotate=None):
+        """Get the text format of the expression.
+
+        Parameters
+        ----------
+        show_meta_data : bool
+            Whether to include meta data section in the text
+            if there is meta data.
+
+        annotate: Optional[Object->str]
+            Optional annotate function to provide additional
```

Comment:
Optionally

Reply:
Optional is a type annotation in python3

Reply:
yeah, I was talking about "Optional annotate function to ..." where "Optional" should be an adverb.

Reply:
ah i see, thanks for the catch!